thirdeye/thirdeye-frontend/app/pods/manage/alerts/performance/route.js

```diff
@@ -69,7 +92,7 @@ const fillAppBuckets = (allApps, validGroups) => {
     let associatedGroups = validGroups.filter(group => group.application.toLowerCase().includes(app.application));
     if (associatedGroups.length) {
       let uniqueIds = Array.from(new Set([].concat(...associatedGroups.map(group => group.emailConfig.functionIds))));
-      if (demoMode) {
+      if (isDemoMode) {
```

Comment:
nit: is there any way we can achieve the same result without the use of "global" boolean that's being checked in multiple places. We should attempt to make it more maintainable

Reply:
We can discuss. For now its just a way to load less data in dev mode.